노현호/스택_큐/주식가격.js

```diff
@@ -0,0 +1,22 @@
+//3가지 고려하기
+//현재 위치 주식 가격이 다음 주식 보다 낮을 경우
+//다음 주식 보다 클 경우
+//마지막 자리 일경우
+
+
+function solution(prices) {
+    const answer = [];
+    let len = prices.length;
+
+    for (let i = 0; i < len; i++) {
+        let time = 0;
+        for (let j = i + 1; j <= len - 1; j++) {
```

Comment:
첫번째 for문에서는 `<`를, 두번째 for문에서는 `<=`를 쓰셨는데 특별한 의도가 있는 걸까요? 통일성을 위해 아래와 같은 방법을 제안드립니다!

```suggestion
        for (let j = i + 1; j < len; j++) {
```

Reply:
2중 for문 돌면서 j는 i 이후의 값만 비교하면 되기 때문에 저렇게 표현하신 거 같네요